app/errors/custom_errors.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module CustomErrors
```

Comment:
This exception let semantically move to `app/operations/ops/exceptions` for example.